src/main/java/org/owasp/wrongsecrets/challenges/kubernetes/Challenge46.java

```diff
@@ -0,0 +1,26 @@
+package org.owasp.wrongsecrets.challenges.kubernetes;
+
+import com.google.common.base.Strings;
+import org.owasp.wrongsecrets.challenges.FixedAnswerChallenge;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+/** This challenge is about having a secrets injected via Vault template. */
+@Component
+public class Challenge46 extends FixedAnswerChallenge {
+
+  private final Vaultpassword vaultPassword;
```

Comment:
Instead of teh VAultpassword, can you please create a new entry? otherwise the answer is the same as challenge7

Reply:
@commjoen done